src/app/components/login/login.blp

```diff
@@ -1,142 +1,100 @@
 using Gtk 4.0;
 using Adw 1;
 
+
 template LoginWindow : Adw.Window {
   default-width: 360;
   default-height: 100;
 
-  WindowHandle {
-    Box {
-      margin-start: 4;
-      margin-end: 4;
-      margin-top: 4;
-      margin-bottom: 4;
-      orientation: vertical;
+  Box {
+    hexpand: true;
+    margin-bottom: 24;
+    orientation: vertical;
 
-      Button close_button {
-        valign: center;
-        receives-default: true;
-        halign: end;
-        has-frame: false;
-        width-request: 22;
-        height-request: 22;
-        icon-name: "window-close-symbolic";
+    Adw.HeaderBar {
+      [title]
+      Label {}
```

Comment:
there's probably a better way to do this...

Reply:
Nicely done, I like it! 🚀

Is the text beneath the title left-aligned? It would probably look more aesthetically pleasing if it was center-aligned like the rest of the window text.

I don’t have an easy-to-remember Spotify username as it was automatically generated when we created our family subscription, but I’m able to log in with my email. We should specify that that’s possible as well.

I also have some small string changes I’d make, but all of this is simple enough for me to do it myself and open a PR instead of listing them and making someone else do it 😅

Reply:
Aaaaand I didn't even manage to *build* it. Opening an issue instead :slightly_smiling_face:

Reply:
Thanks! Yeah I should have involved more formally in a review. Thanks for the upcoming/PR issue I'll have a look. Oh and if you want help with building the app we can look it eventually, here or on matrix!